src/raw/mod.rs

```diff
@@ -1492,7 +1498,9 @@ impl RawTableInner {
     const fn new() -> Self {
         Self {
             // Be careful to cast the entire slice to a raw pointer.
-            ctrl: unsafe { NonNull::new_unchecked(Group::static_empty() as *const _ as *mut u8) },
+            ctrl: unsafe {
+                NonNull::new_unchecked(Group::static_empty().as_ptr() as *const u8 as *mut u8)
+            },
```

Comment:
I wanted to make this `cast_mut().cast()` but `cast_mut` would require bumping the MSRV, and not really sure how folks felt about that.

Reply:
Rust 1.63 is over 2 years old. Feel free to bump MSRV all the way up to 1.80 (oldstable).

Reply:
I'm gonna do that, then.